src/components/full-text-search/search-form.tsx

```diff
@@ -0,0 +1,278 @@
+import queryString from "query-string";
+import {
+  Stack,
+  Input,
+  InputGroup,
+  InputLeftAddon,
+  InputRightElement,
+  Button,
+} from "@chakra-ui/react";
+import { CalendarIcon } from "@chakra-ui/icons";
+import { useState, useEffect, FormEvent } from "react";
+import { useRouter } from "next/router";
+import { useGetData } from "./hooks/use-get-data";
+import { SelectInput } from "@bit/limebit.chakra-ui-recipes.select-input";
+
+export interface FormParams {
+  contentQuery?: string | null;
+  factionIdQuery?: string | null;
+  politicianIdQuery?: string | null;
+  positionShortQuery?: string | null;
+  fromDate?: string | null;
+  toDate?: string | null;
+}
+
+export interface Faction {
+  id: string;
+  fullName: string;
+  abbreviation: string;
+}
+
+export interface Politician {
+  id: string;
+  firstName: string;
+  lastName: string;
+}
+
+export const SearchForm: React.FC<FormParams> = () => {
```

Comment:
Vllt kannst du allgemein nochmal checken, dass wir hier die richtigen input types nutzen, bspw.
`type="search"`
siehe: https://www.w3schools.com/html/html_form_input_types.asp

Bei mir zeigt chrome nämlich automatisch z.B. das lastpass icon an, das macht er aber nicht wenn der type="search" gesetzt ist. Über den type und den form name funktionieren z.B. auch die automatischen prefills von formularen, das sollte in dem fall auch vermieden werden
![Bildschirmfoto 2020-11-18 um 07 04 42](https://user-images.githubusercontent.com/12813089/99491639-66a18400-296c-11eb-8bdd-9db3cb7d6c1b.jpg)

Reply:
Kannst du gucken, wie es in der neuen Version ist? `type=search` wollte ich nicht unbedingt benutzen, weil Safari und Chrome sonst ihre eigenen 'x'-Buttons einfügen

Reply:
mh okay. gibt es vllt eine property, mit der man das verhindern kann? sowas wie autofill=false oder so?

Reply:
bei mir ist es noch da. Probier mal `background-image: none;` auf der component. bei mir ist das lastpass icon dann weg. vllt geht das auch mit den anderen icons?